### ISSUE TITLE
[UX] Wait a little before considering the room as read

### ISSUE THREAD

**Issue by MightyCreak** [vector-im/element-web#6386]
### Description

It seems like the rooms are considered as read as soon as focus is on them. But sometime, I happen to click a bit too quickly on a room and click on the one I really wanted to see, but the first room is marked as read already and if I go back, I don't know where I was anymore. Another repro is that I receive a notification saying I have a message in room "B", I <kbd>Alt</kbd><kbd>Tab</kbd> to Riot and the current room gets read even though I stayed there a few seconds before going to the room "B".

Something like 3 seconds before the room is effectively marked as read would be great.

### Version information

- **Platform**: desktop

For the desktop app:

- **OS**: all
- **Version**: 0.13.5

**Comment by MightyCreak** [comment]
This issue is loosely linked to #4406

**Comment by turt2live** [comment]
This sounds like https://github.com/vector-im/riot-web/issues/3653

**Comment by MightyCreak** [comment]
I haven't found this issue when I search, but it's not exactly the same. It seems quite technical because they're talking 100ms, and here I'm talking about a more human response time, around 3s.

The idea is that I don't want the room I just focused on to be marked as read even is I stay on it just 1s. 1s is not enough to read a line of text, unless the text is very short, but then 3s is not that long for acknowledging that the user opened the room, read the short text and considered it read.

**Comment by MTRNord** [comment]
@MightyCreak I personaly think 3s would be way to long. I would need to wait 3 seconds atleast when clicking trough every room that would be much longer than i currently need. I think a Option I  the settings Page on how long it should stay would be a lot better.

**Comment by MightyCreak** [comment]
I'm really not fond of settings for everything. Doing so, the settings page ends up being a huge mess of tiny options that no one ever go through because it's too abstract for them. And I don't even talk about Q&A here. Each option has to be tested in different value, AND combined together because it might create a conflict. Honestly, it's a huge nightmare.

A good UX should limit the number of settings.

I personally don't go through each room just to mark them as read. I don't _need_ to have them all read all the time. At first, I was doing that, but at some point I figured that it's pointless to try to be up to date with an IRC chan with 1000 people in it. So I simply changed the notification priorities of the room that are less important to me.

All that to say that, 1) 3s is an example, it could be 2s, maybe 1s, but it has to be "human". If you're goal is to click as fast as possible through the unread rooms in order to mark them read, maybe there's a better solution. And maybe the right question to ask ourself is "is it what most people do?". And 2) we could add a "Mark as read" in the context menu of the room. I'm not fond of too cluttered context menu, but if there's a need for this feature, then it needs to be accessible somehow.
